src/api/files/service.test.js

```diff
@@ -10,9 +17,11 @@ import * as repository from '~/src/api/files/repository.js'
 import {
   checkExists,
   ingestFile,
-  getPresignedLink
+  getPresignedLink,
+  persistFile
 } from '~/src/api/files/service.js'
 import { prepareDb } from '~/src/mongo.js'
+import 'aws-sdk-client-mock-jest'
```

Comment:
Should we create a `jest.setup.cjs` file and import this once for all files?

Reply:
The CDP templates ([backend](https://github.com/DEFRA/cdp-node-backend-template), [frontend](https://github.com/DEFRA/cdp-node-frontend-template/tree/main)) both have ESLint type checking now

Not for this PR, but would be good to check this stuff is wired up correctly

Reply:
I'll leave this one for now, I'd rather the explicit import since we only have one file so we don't override any mocking behaviour unexpectedly.